graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -251,6 +253,12 @@ public LdapEntry search(LdapNetworkConnection connection,
                     }
                 }
             }
+        } catch (Exception e) {
+            LOG.warn(
```

Comment:
I'm a little bit uneasy about just swallowing the exception. Could we at least create a system notification as well so that people who don't parse the Graylog logs every day will have an idea that something went wrong?